plugins/ti.alloy/hooks/alloy.js

```diff
@@ -152,9 +152,10 @@ exports.init = function (logger, config, cli, appc) {
 						cmd.map(function(a) {
 							if (/^[^"].* .*[^"]/.test(a)) return '"' + a + '"'; return a;
 						}).join(' ') + '"'].join(' ')], {
-						stdio: 'inherit',
-						windowsVerbatimArguments: true
-					});
+							stdio: 'inherit',
+							windowsVerbatimArguments: true
+						}
+					);
```

Comment:
This is driving me crazy since 5+ years. @ewanharris Do you know why this randomly changes between Alloy versions and even builds?

Reply:
this file could be in .gitignore by default right?

Reply:
Good idea! Can you do a ticket / PR for that? Looow prio!

Reply:
I think alloy/appc-cli update it on build. Hook was [linted a couple months back](https://github.com/appcelerator/alloy/commit/5980ddd8cdcc3c732c4a5cb7e8b4e40165647c0c#diff-6487ccfa4909144bbc33134e4d85bdddR155), so best guess is @Topener needs to update his Alloy or appc-cli version. But I do kinda agree that it should maybe be in the .gitignore as it causes needles noise in commits

Reply:
@ewanharris I do run Alloy 1.11 though!